vulture_os/vulture_os/settings.py

```diff
@@ -182,20 +182,19 @@
 
 
 # Internationalization
-# https://docs.djangoproject.com/en/2.0/topics/i18n/
+# https://docs.djangoproject.com/en/4.2/topics/i18n/
 
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'UTC'
 USE_I18N = True
-USE_L10N = True
```

Comment:
same question: why is it necessary to remove this line?

Reply:
It is deprecated and throw a warning on django check routine : https://github.com/adamchainz/django-upgrade#use_l10n